lib/workers/repository/update/branch/index.ts

```diff
@@ -698,9 +699,16 @@ export async function processBranch(
           ' - you rename this PR\'s title to start with "rebase!" to trigger it manually';
         content += '\n\nThe artifact failure details are included below:\n\n';
         // TODO: types (#7154)
+
+        const stringifyArtifactError = (err: ArtifactError): string => {
+          const msg: (string | undefined)[] = [];
+          msg.push(err.message, err.stderr, err.stdout);
```

Comment:
1. I'm worried about stdout length
2. I don't think we can just concatenate them without explanation
3. This would likely be out of order anyway, because usually stdout would appear first and stderr at the end

Reply:
1. what about the suggestion above, we could have collapsible section for the streams 
2. if we use collapsibles we can cover this with the summary tag
3. they are pushed one by one by the order they appear e.g.
<img width="245" alt="image" src="https://user-images.githubusercontent.com/97394622/182540425-d7d5a00b-555a-40bc-9f48-feba6da97d0a.png">

Reply:
from what i saw, collapsible sections does not work inside code blocks.